Python/Product/TestAdapter.Executor/Services/ExecutorService.cs

```diff
@@ -88,17 +87,27 @@ sealed class TestRedirector : Redirector {
                 _debugSecret,
                 _debugPort.ToString(),
                 GetDebuggerSearchPath(_projectSettings.UseLegacyDebugger),
-                String.Format("--junitxml={0}", outputfile)
             };
 
-            foreach (TestCase test in tests) {
-                var executionTestPath = test.GetPropertyValue<string>(Pytest.Constants.PytestIdProperty, default);
-                if (String.IsNullOrEmpty(executionTestPath)) {
-                    Debug.WriteLine("Pytest execution path missing for testcase {0}", test.FullyQualifiedName);
-                    continue;
+            // For a small set of tests, we'll pass them on the command
+            // line. Once we exceed a certain (arbitrary) number, create
+            // a test list on disk so that we do not overflow the 
+            // 32K argument limit.
+            var testIds = tests.Select(t => t.GetPropertyValue<string>(Pytest.Constants.PytestIdProperty, default));
+            if (testIds.Count() > 5 ) {
+                var testListFilePath = TestUtils.CreateTestList(testIds);
+                arguments.Add(testListFilePath);
+            }
+            else {
+                arguments.Add("blankTestList");
```

Comment:
is this a magic string recognized by pytest? or you're counting on it trying to open the file and ignoring it since it doesn't exist? worth a comment in the code.

Reply:
ya, just needed a filename to ignore.. i'll add a comment